packages/react-compose/src/utils.ts

```diff
@@ -29,6 +35,16 @@ export const defaultComposeOptions: ComposePreparedOptions = {
   resolveSlotProps: () => ({}),
 };
 
+const mergeClasses = (classes1: ClassDictionary | undefined, classes2: ClassDictionary | undefined) => {
+  const result = { ...classes1 };
+
+  if (classes2) {
+    Object.keys(classes2).forEach(name => (result[name] = cx(result[name], classes2[name])));
+  }
+  console.log(result);
+  return result;
+};
+
 export function mergeComposeOptions(
```

Comment:
no tests?

Reply:
This is in a separate PR (with tests :)

Reply:
https://github.com/microsoft/fluentui/pull/13101